docs/introduction/QuickStart.mdx

```diff
@@ -0,0 +1,177 @@
+
+### Create a React Project
+
+We'll start by creating an empty React project. You can do this in one of two ways: creating a local project on your own computer with the
+Create-React-App tool, or creating a project in the CodeSandbox web app.
+
+#### Create a Local Project with Create-React-App
+
+[Create-React-App](https://create-react-app.dev/) is the official tool for creating new React projects.
+
+Create a new React project named `redux-employee-example` by running this command:
+
+```bash
+npx create-react-app redux-employee-example
+```
+
+Then, install the required Redux packages, using either the NPM or Yarn package manager:
+
+<Tabs
+  defaultValue="npm"
+  values={[
+    { label: 'NPM', value: 'npm', },
+    { label: 'Yarn', value: 'yarn', }
+  ]
+}>
+
+<TabItem value="npm">
+
+```bash
+npm add @reduxjs/toolkit react-redux
+```
+
+</TabItem>
+
+<TabItem value="yarn">
+
+```bash
+yarn add @reduxjs/toolkit react-redux
+```
+
+</TabItem>
+
+</Tabs>
+
+#### Create an Online Project with CodeSandbox
+
+[CodeSandbox.io](https://codesandbox.io) is an online IDE that lets you create entire web applications in your browser, without needing to
+install anything locally.
+
+Create a new React project sandbox by browsing to the [CodeSandbox "Create Sandbox" page](https://codesandbox.io/s/) and selecting the official React template.
+
+Once the sandbox has been created, click the "Add Dependency" button on the left, search for `@reduxjs/toolkit`, add it, and then do the same for `react-redux`.
+
+### Create a Redux Store
+
+The React project starts with a single `<App>` component that shows some "hello" text on the page. Our first goal is to create a Redux store instance
+that holds some data inside, and update the `<App>` component to read that data and show it instead.
+
+To create a Redux store, start by updating the `index.js` file to import the `configureStore` function from the `@reduxjs/toolkit` package.
+
+```diff
+import React from "react";
+import ReactDOM from "react-dom";
++import {configureStore} from "@reduxjs/toolkit";
+
+import App from "./App";
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(<App />, rootElement);
+```
+
+In order to actually create a Redux store, we need to define our first "reducer" function. Reducers are functions that know how to
+calculate a new state value when asked. A Redux store must always be given a single reducer function when it is created, which we call the
+"root reducer" to say that it is the starting point for any other reducer functions inside.
+
+For now, we can create a small root reducer function that always returns a string, like this:
+
+```diff
+import React from "react";
+import ReactDOM from "react-dom";
+import {configureStore} from "@reduxjs/toolkit";
+
+import App from "./App";
+
++function rootReducer() {
++    return "Hello Redux!";
++}
+
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(<App />, rootElement);
+```
```

Comment:
I like the idea of introducing the root reducer as soon as the store is configured, but I think it might be more clear to a beginner what the reducer is used for if there's a more real-world example state here. For example, it could have `return { firstName: "Homer", lastName: "Simpson" }` and then later add actions/reducers/slices for updating the name fields, showing the relationship between the initial state and how Redux stores are updated as they're developed.

Reply:
My intent is to have the first step simply be `() => "Hello Redux"` + `const helloText = useSelector(state => state)`, and get that to show up on screen instead of the initial text in `<App>`.  Then, once they've seen the initial connection sequence, we can move on to filling out actual app logic.